server/.env.test

```diff
@@ -6,9 +6,10 @@ PG_DATABASE_URL=postgres://twenty:twenty@localhost:5432/test?connection_limit=1
 # the URL of the front-end app
 FRONT_BASE_URL=http://localhost:3001
 # random keys used to generate JWT tokens
-ACCESS_TOKEN_SECRET=secret_jwt 
+ACCESS_TOKEN_SECRET=secret_jwt
 LOGIN_TOKEN_SECRET=secret_login_tokens
-REFRESH_TOKEN_SECRET=secret_refresh_token  
+API_TOKEN_SECRET=secret_api_tokens
```

Comment:
@martmull Sorry I've miss to tell you, but we need to add the new properties in the environment validation [file](https://github.com/twentyhq/twenty/blob/main/server/src/integrations/environment/environment.validation.ts).

Reply:
think i have done it -> https://github.com/twentyhq/twenty/pull/1967/files#diff-b5ad9883b32f42c1a75704c4c5559b316afba735a9eaf8be074044dcf86ae711